keyboards/lily58/keymaps/barabas/keymap.c

```diff
@@ -0,0 +1,197 @@
+/*
+Copyright 2020 Barabas Raffai
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+#ifdef PROTOCOL_LUFA
+#    include "lufa.h"
+#    include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+#    include "ssd1306.h"
+#endif
```

Comment:
This shouldn't be needed. 
```suggestion
```

Reply:
I've removed it. FYI, this exists in many other keymaps as well. I copied that from the default keymap for lily58.